devito/cgen_wrapper.py

```diff
@@ -1,6 +1,6 @@
 import ctypes
 
-from cgen import *
+from codepy.cgen import *
```

Comment:
Are you sure you are not moving backward here? `codepy.cgen` is deprecated as seen [here](https://github.com/inducer/codepy/blob/master/codepy/cgen/__init__.py)